### ISSUE TITLE
Wrong hyperlinks when source is in multiple .yml files

### ISSUE THREAD

**Issue by nirakon** [innoverio/vscode-dbt-power-user#51]
I have

```
├── dbt_project.yml
└── models
    └── staging
        └── source_1
        |  └── source_1.yml
        └── source_2
           └── source_2.yml

```

inside source_1.yml

```
version: 2

sources:
  - name: bq_dataset_1
    database: project_id_1
    tables:

      - name: unique_table_name_1
```

inside source_2.yml

```
version: 2

sources:
  - name: bq_dataset_1
    database: project_id_1
    tables:

      - name: unique_table_name_2
```

Inside a model I have `{{ source('bq_dataset_1', 'unique_table_name_2') }}` When using "go to definition" on `'bq_dataset_1'` it's always linked to source_1.yml. Also "go to definition" on `'unique_table_name_2'` not working at all.

The problem also replicates in sidebar extension (Parent Models, Children Models)

**Comment by mdesmet** [comment]
Hi Ivan, I will release a new version this week fixing this issue.

**Comment by mdesmet** [comment]
HI Ivan, this should be resolved in 0.5.0. Let me know if it works for you.

**Comment by nirakon** [comment]
Yes, it's working. Thanks a lot!
